dotnet-471-dev-pack/README.md

```diff
@@ -0,0 +1,107 @@
+# Habitat package: dotnet471
+
+## Description
+
+This package is for the .NET 4.7.1 framework developer pack from microsoft. It provides 4.7.1 reference assemblies as well as the SDK needed to support building .NET 4.7.1 applications
+
+## Maintainers
+
+The Habitat Maintainers humans@habitat.sh
+
+## Type of Package
+
+Binary
+
+## Usage
+
+Add core/dotnet471 to pkg_build_deps then leverage nuget and msbuild to build and deploy your application. For an example see mwrock's asp full framework MVC blog post: https://www.habitat.sh/blog/2017/08/Habitat-plans-for-net-full-iis-apps/
+
+## Notes

```

Comment:
This heading has no content, it can possibly be deleted.

Reply:
"Notes" heading removed